mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 # Project information
-site_name: AVD
+site_name: ""
```

Comment:
This breaks tab titles in browser

<img width="255" height="59" alt="Screenshot 2025-07-17 at 11 39 31" src="https://github.com/user-attachments/assets/1c1f899c-1df8-4eb1-af0f-3c9512d11ad8" />
<img width="276" height="56" alt="Screenshot 2025-07-17 at 11 39 38" src="https://github.com/user-attachments/assets/e6b203bc-5d02-4574-8a0f-d0e076a6aee1" />